.github/workflows/gt4py-tox.yml

```diff
@@ -20,6 +20,11 @@ jobs:
     strategy:
       matrix:
         python-version: [3.8, 3.9.7]
+        backends: [internal-cpu, with-externals-cpu]
+        exclude:
+        - python-version: 3.8
+          backends: internal-cpu
```

Comment:
the other 2 are running **all** backends, here we are only testing if all works in case external backends are not there. Unless you expect that there could be a case where
- 3.9 with only internal works
- 3.8 with all works
- 3.8 with only internal breaks
That would be a python version dependent code that relies on dace being installed.

Reply:
Ok, I see. It makes sense to save CI hours.